src/test/java/uk/gov/hmcts/reform/jobscheduler/jobs/HttpCallJobTest.java

```diff
@@ -49,21 +53,25 @@ public void execute_should_send_http_request_specified_in_job_details() {
         given(context.getJobDetail())
             .willReturn(newJob(HttpCallJob.class).withIdentity("id", "group").build());
 
+        Map<String, String> headers = new HashMap<>();
+        headers.put("ServiceAuthorization", "some-token");
```

Comment:
What happens to this header sent by the client? does it get replaced?
I think it would be good to use some other generic header here and have a separate test (with nice name) that tests what happens in such a case.

Reply:
Yes, `ServiceAuthorization` gets replaced. I'll add the test with another header.